client/src/pages/ContactsPage/ContactsPage.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react'
+import {Box, Link, Grid, Stack, Container, Typography} from "@mui/material"
+
+import './Contacts.scss'
+
+import {Form ,Social,Map} from '../../components'
+
+
+
+const ContactsPage = () => (
+    <Stack direction='column' maxWidth='1200px' justifyContent='center' m =' 50px auto 100px auto' id="contacts">
+        <Container alignItems='center' className='contacts-header'><Typography textAlign='center' variant="h4">Contact
```

Comment:
className="contacts-header"

i guess this class doesnt exist in the style sheet